code/game/machinery/clonepod.dm

```diff
@@ -623,14 +629,36 @@
 
 /obj/machinery/clonepod/emag_act(user)
 	. = ..()
-	eject_clone(TRUE)
+	malfunction()
 	return TRUE
 
 /obj/machinery/clonepod/emp_act(severity)
-	if(prob(50))
-		eject_clone(TRUE)
+	if(prob(100 / (severity * emp_resistance)))
+		malfunction()
 	return ..()
 
+/obj/machinery/clonepod/proc/malfunction()
+	if(clone)
+		var/datum/mind/patient_mind = locateUID(patient_data.mindUID)
+		patient_mind.transfer_to(clone)
+		clone.grab_ghost()
+		var/message
+		message += "<b>Agony blazes across your consciousness as your body is torn apart.</b><br>"
+		message += "<i>Is this what dying is like? Yes it is.</i>"
+		to_chat(clone, "<span class='warning'>[message]</span>")
+		SEND_SOUND(clone, sound('sound/hallucinations/veryfar_noise.ogg', 0, 1, 50))
+		sleep(40)
+		new /obj/effect/gibspawner/generic(get_turf(src), clone.dna)
+		new /obj/effect/gibspawner/generic(get_turf(src), clone.dna)
+		new /obj/effect/gibspawner/generic(get_turf(src), clone.dna)
```

Comment:
is there some way to spawn the species organs and limbs? having it throw gibs around is cool and all, but imagine it throwing arms, hearts, and legs out

Reply:
I don't believe we have species-specific gibspawners. That would be really cool to see, though.